packages/css-algorithms/scripts/stylelint-css-algorithms.js

```diff
@@ -0,0 +1,51 @@
+var stylelint = require("stylelint")
+var chalk = require( "chalk")
+
+var ruleName = "plugin/css-algorithms"
+
+var messages =  stylelint.utils.ruleMessages(ruleName, {
+	expected: "Expected ...",
+})
+
+module.exports = stylelint.createPlugin(ruleName, function( options ) {
+	return function( cssRoot, result ) {
+
+		var validOptions = stylelint.utils.validateOptions( 
+			result, 
+			ruleName, 
+			{
+				actual: options,
+				possible: {
+					'name': 'a-space-children',
+					'allowed-properties': [ 'margin-top', 'margin-left', 'display', 'flex-wrap', '--a-space-children-spacer' ]
+				}
+			}
+		);
+
+		if (!validOptions) { 
```

Comment:
Passing but would recommend logging/messaging regarding invalid/unset options.

Reply:
I still need to actually write some validation methods here – I believe stylelint does log a message but will check. Thanks!